sources/engine/Stride.Engine/Profiling/DebugTextSystem.cs

```diff
@@ -64,14 +88,20 @@ public void Print(string message, Int2 position, Color4 color)
         /// </summary>
         public Color4 TextColor { get; set; } = Color.LightGreen;
 
+        /// <summary>
+        /// Sets or gets the time that messages will stay on screen by default.
+        /// </summary>
+        public TimeSpan DefaultOnScreenTime { get; set; } = TimeSpan.FromSeconds(1);
+
         /// <summary>
         /// Sets or gets the size of the messages queue, older messages will be discarded if the size is greater.
         /// </summary>
         public int TailSize { get; set; } = 100;
 
         public override void Update(GameTime gameTime)
         {
-            overlayMessages.Clear();
+            // if you keep this, the timmers would not work
+            //overlayMessages.Clear();
```

Comment:
Feel free to remove this given that it's not applicable anymore

Reply:
Removed, shall we keep the Update method even if it's empty anyways?

Reply:
Nope, you can remove it too